src/main/kotlin/com/zepben/evolve/services/network/NetworkServiceComparator.kt

```diff
@@ -753,6 +756,24 @@ class NetworkServiceComparator @JvmOverloads constructor(
             compareConductingEquipment()
         }
 
+    private fun compareGroundingImpedance(source: GroundingImpedance, target: GroundingImpedance): ObjectDifference<GroundingImpedance> =
+        ObjectDifference(source, target).apply {
+            compareEarthFaultCompensator()
+
+            compareValues(
+                GroundingImpedance::x
+            )
+        }
+
+    private fun ObjectDifference<out EarthFaultCompensator>.compareEarthFaultCompensator(): ObjectDifference<out EarthFaultCompensator> =
```

Comment:
todo: check all functions for alphabetical ordering and correct package location

Reply:
Moved